technical/indicators.py

```diff
@@ -845,3 +845,42 @@ def return_on_investment(dataframe, decimals=2) -> DataFrame:
     dataframe['roi'] = roi
     
     return dataframe
+
+def td_sequential(dataframe):
+    """
+    TD Sequential
+    Author(Freqtrade): MichealReed
+    Original Author: Tom Demark
+
+
+    :param dataframe: dataframe
+    :return: TD Sequential:values - to +
+    """
+    """
+    TD Sequential
+    How to Buy/Sell Cryptocurrency with TD Sequential indicator
+    https://hackernoon.com/how-to-buy-sell-cryptocurrency-with-number-indicator-td-sequential-5af46f0ebce1
+    """
+    import pandas as pd
+    import numpy as np
```

Comment:
At first look, i'd say that these imports are not necessary - `np` is not used and DataFrame (the only used funciton) is imported directly.